test/CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ list( APPEND test_input
   testinput/aeronet_cad.dat
   testinput/aeronet_tab.dat
   testinput/imsscf_20191215_c48.nc
-  testinput/bfrPrf_C2E3.2020.349.21.26.R18_0001.0001_bufr
+  testinput/bfrPrf_C2E6.2021.214.12.00.G16_0001.0001_bufr
```

Comment:
is this data uploaded to data repo already?

Reply:
ioda-converters has no ioda-converters-data repository. So, yes, the data are already uploaded.